### PR TITLE
Feature/reactor api

### PR DESCRIPTION
**Pull request by chesterlaw**
@wangdrew

### CODE REVIEW

**Comment by chesterlaw** [comment]
@wangdrew

**Comment by chesterlaw** [comment]
@wangdrew

File: conf/nginx.conf [hunk 1]
```diff
@@ -174,6 +179,25 @@ http {
...
       proxy_pass http://mock_events_servers;
-    }    
+    }
+
+
+    # ice-reactor API
+    location /api/vps/v1/icereactor {
+      access_by_lua_file "checkToken.lua";
+
+      rewrite /api/vps/v1/icereactor/(.*)$ /v2/$1 break;
```
Comment (wangdrew) [inline conf/nginx.conf]: you should start querying `/v2` directly rather than use `/api/vps/v1`. It's fine for now

**Comment by wangdrew** [comment]
if it works, lgtm

**Pull request merged**